src/data/events/events.js

```diff
@@ -37,7 +37,8 @@ const events = [
 		date: getDateTime(2022, 10, 5, 18),
 		location: 'Haines Hall A2',
 		imgFilePath: 'event/2022f-hackschool-banner.png',
-		detailLink: 'https://discord.gg/rup2p6fxA5'
+		detailLink: 'https://discord.gg/rup2p6fxA5',
+		rsvpLink: 'https://fb.me/e/2jIVJgX9j'
```

Comment:
Is there a google form for rsvp'ing, or is the facebook event all we have?

Reply:
The Facebook event is all we have. Do you think it’s even worth linking?

Reply:
yeah..., usually we have the detailLink be the facebook link